nservicebus/errors/automatic-retries.md

```diff
@@ -70,6 +70,10 @@ NOTE: Retrying messages for extended periods of time would hide failures from op
 
 SLR can be configured in several ways.
 
+### Transport transaction requirements
+
+The SLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md) and scheduling message for [delayed-delivery](/nservicebus/messaging/delayed-delivery.md) next time it is picked from the input queue. Aborting the receive operation when transactions are turned off would results in message loss. Therefore SLR cannot be used when transport transactions are disabled.
```

Comment:
> That seems unnecessary to me. Skip? Or say "the next time...".

I would like to leave that because it shows the user that for SLR to work the message has to be delivered to the endpoint second time.